nixos/modules/services/web-apps/zipline.nix

```diff
@@ -0,0 +1,112 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.zipline;
+in
+{
+  meta.maintainers = with lib.maintainers; [ defelo ];
+
+  options.services.zipline = {
+    enable = lib.mkEnableOption "Zipline";
+
+    package = lib.mkPackageOption pkgs "zipline" { };
+
+    settings = lib.mkOption {
+      type = lib.types.attrsOf lib.types.str;
```

Comment:
We should use a submodule type otherwise the module system is not properly merging multiple of those.